assets/stylesheets/layout/_main.scss

```diff
@@ -280,6 +280,13 @@ html.iframed {
 	line-height: 1;
 }
 
+/* Emoji and smileys */
```

Comment:
should we drop https://github.com/Automattic/wp-calypso/blob/master/client/reader/_style.scss#L10 in favor of this?